tests-legacy/PrestaShopBundle/Controller/ControllerTest.php

```diff
@@ -149,8 +149,7 @@ public function getControllersClasses()
 
     protected function declareRequiredConstants()
     {
-        $configDirectory = __DIR__ . '/../../../app/config';
-        $configuration = require_once $configDirectory . '/parameters.php';
+        $configuration = require_once _PS_CACHE_DIR_ . 'appParameters.php';
```

Comment:
Why change this include? What if the cache folder has been emptied? Using the file in config seems safer no?

Reply:
parameters.php doesn't take in consideration what have be done in boostrap.
the `appParemeters.php` is the computed cache between `paremeters.yml`, and `parameters.php`, many configuration are missing if we don't use the right file. (needed for the utf8mb4 pull request)